src/Forms/ElementalAreaField.php

```diff
@@ -111,8 +111,11 @@ public function FieldHolder($properties = array())
     public function getSchemaDataDefaults()
     {
         $schemaData = parent::getSchemaDataDefaults();
-        $pageId = ($this->getArea() && ($page = $this->getArea()->getOwnerPage())) ? $page->ID : null;
+
+        $area = $this->getArea();
+        $pageId = ($area && ($page = $area->getOwnerPage())) ? $page->ID : null;
         $schemaData['page-id'] = $pageId;
+        $schemaData['elemental-area-id'] = $area ? $area->ID : null;
```

Comment:
Right so if ElementEditor gets the ID from here we can definitely cast as int

Reply:
related proptypes are all `number.isRequired` now :)